packages/react/src/radio-group/RadioGroup.test.tsx

```diff
@@ -154,26 +154,67 @@ describe('<RadioGroup />', () => {
     expect(indicator).to.have.attribute('data-required', '');
   });
 
-  it('should set the name attribute on the input', async () => {
-    await render(<RadioGroup name="radio-group" />);
+  it('should set the name attribute on the input only when a value is selected', async () => {
+    await render(
+      <RadioGroup name="radio-group">
+        <Radio.Root value="a" data-testid="radio" />
+      </RadioGroup>,
+    );
     const group = screen.getByRole('radiogroup');
+    const radio = screen.getByTestId('radio');
+
+    expect(group.nextElementSibling).to.not.have.attribute('name');
+
+    act(() => {
+      radio.click();
+    });
     expect(group.nextElementSibling).to.have.attribute('name', 'radio-group');
   });
 
+  it('should return null when no radio is selected (matching native behavior)', async ({
+    skip,
+  }) => {
+    if (isJSDOM) {
+      // FormData is not available in JSDOM
+      skip();
+    }
```

Comment:
`it.skipIf(isJSDOM)` is now a possible method
Not all tests were updated to this format yet, but we've been using this going forward